Userland/Libraries/LibCore/Stream.h

```diff
@@ -24,6 +24,48 @@
 
 namespace Core::Stream {
 
+template<DerivedFrom<Core::Stream::Stream> T>
```

Comment:
Do we need to specify the full namespace here? We're already in `
namespace Core::Stream`.